src/main/ruby/core/complex.rb

```diff
@@ -303,7 +303,7 @@ def rationalize(eps = nil)
   def to_s
     result = real.to_s
 
-    if imag.kind_of?(Float) ? !imag.nan? && imag.signbit? : imag < 0
+    if imag < 0 or imag.equal?(-0.0)
```

Comment:
I don't know if we have a Rubocop rule for this, but I thought we settled on `||` for simple boolean expressions like this.